lib/Doctrine/Common/Cache/RedisCache.php

```diff
@@ -44,7 +44,11 @@ class RedisCache extends CacheProvider
      */
     public function setRedis(Redis $redis)
     {
-        $redis->setOption(Redis::OPT_SERIALIZER, $this->getSerializerValue());
+        // The redis object is injected from an external service
+        // Serializer option should be set outside
+        // We should not force the serializer option here
+        // This code is left here for reference only
```

Comment:
Thanks stof, I will wait for the response from the maintainer then if the change is ok I will remove the reference code as you suggest.